frontend/src/api/ExecutionRequestApi.tsx

```diff
@@ -91,7 +91,7 @@ const CSVDownloadSchema = z.object({
 
 const RawDatasourceRequestSchema = z.object({
   id: z.string(),
-  type: z.enum(["TemporaryAccess", "SingleExecution"]),
+  type: z.enum(["TemporaryAccess", "SingleExecution", "GetSQLDump"]),
```

Comment:
Now that I see it for the fifth time, maybe just call it `SQLDump` without the get? 😁

Reply:
Of course! Updated it!